dotcom-rendering/src/components/TextBlockComponent.tsx

```diff
@@ -193,20 +192,20 @@ const styles = (format: ArticleFormat) => css`
 		display: inline-block;
 		content: '';
 		border-radius: 50%;
-		height: ${space[3]}px;
-		width: ${space[3]}px;
+		height: ${remSpace[3]}px;
+		width: ${remSpace[3]}px;
 		background-color: ${palette.neutral[86]};
-		margin-left: -${space[5]}px;
-		margin-right: ${space[2]}px;
+		margin-left: -${remSpace[5]}px;
+		margin-right: ${remSpace[2]}px;
 	}
 
 	/* Subscript and Superscript styles */
 	sub {
-		bottom: -0.25em;
+		bottom: -${remSpace[1]}rem;
```

Comment:
Was this `em` before?

Reply:
It was yes but I don't think it needs to be? If everything is rem I think this will scale correctly?

Reply:
I _think_ in the particular case of sub and superscript we want them scale according to the font size of their ancestor and not the base font size (as with rem). But it might be worth trying both out and checking how they compare visually.

Someone on the internet [said the same thing](https://forum.freecodecamp.org/t/em-vs-px-things/246036#:~:text=ems%20are%20useful%20for%20things,%3A%200.75em%7D%20or%20whatever.)

Reply:
I can see why because the sub is relative to the thing it's a sub of but if that thing is also scaling according to the base font size doesn't it result in the same behaviour? I'll check :D

Reply:
Regarding checking visually, it would be great to make them closest to existing glyphs, eg. `²` (those should look identical to OpenType features which we don’t use, but could in Full subsets, but that’s a long story and much work)

Reply:
ok leaving as em :D